docs/reference/releases/rev156_136.md

```diff
@@ -0,0 +1,50 @@
+# Revision 156/136
+<sub>Thursday, July 4, 2024</sub>
+
+Dear community,
+
+We are glad to report that we have just released a new updated version for Charmed ZooKeeper on the `3/stable` channel, upgrading its revision from 126 to 136. 
+The release of a new version was promoted by the need of backporting some features that 
+should provide increased robustness and resilience during operation as well as smaller workload upgrades and fixes. See the technical notes for further information.
+
+Please reach out should you have any question, comment, feedback or information. You can find us here in [Matrix](https://matrix.to/#/#charmhub-data-platform:ubuntu.com) or also on [Discourse](https://discourse.charmhub.io/).
+
+## Features
+
+* [DPE-3726] Workload upgrade to 3.8.4-ubuntu0
+* [DPE-4183] (backport) fix: only handle quorum removal on relation-departed
```

Comment:
It's not a new feature, but rather a bug fix. Consider creating a bugfix section for this and the next one.